src/MainWindow.cpp

```diff
@@ -250,10 +250,13 @@ void MainWindow::OpenAbout(void)
 		"Raefaldhi Amartya Junior",
 		"Thomas Schmidt",
 		"waddlesplash",
+		"Tanausú Gómez (Spanish translation)",
```

Comment:
Instead of adding yourself under the developers, I using AddSpecialThanks(). It's used similar to the AddAuthors().

Reply:
I made the proposed changes.
Thanks for your work. Let's merge!